src/plugin/Makefile.in

```diff
@@ -100,7 +100,9 @@ libdmtcp_PROGRAMS = $(d_libdir)/libdmtcp_alloc.so$(EXEEXT) \
 	$(d_libdir)/libdmtcp_timer.so$(EXEEXT)
 subdir = src/plugin
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
```

Comment:
Ditto. And it would be nice to set the CFLAGS variable at top level, instead of spreading out some distributed code for eaach directory.